taipy/common/config/_serializer/_base_serializer.py

```diff
@@ -29,6 +29,19 @@ class _BaseSerializer(object):
     """Base serializer class for taipy configuration."""
 
     _GLOBAL_NODE_NAME = "TAIPY"
+    _SERIALIZABLE_TYPES = [
+        "bool",
+        "str",
+        "int",
+        "float",
+        "datetime",
+        "timedelta",
+        "function",
+        "class",
+        "SCOPE",
+        "FREQUENCY",
```

Comment:
This must be removed and dynamically added from the core package. For instance, it could be 
declared in a `_SUB_TYPES ` variable implemented in the `Section` class, and updated dynamically by the `_inject_section` function (in `common/config/__init__.py`) when registering the Section.